modules/remove-central-attached-pv-overview.adoc

```diff
@@ -3,7 +3,7 @@
 // * upgrading/upgrade-operator.adoc
 :_mod-docs-content-type: CONCEPT
 [id="remove-central-attached-pv-overview_{context}"]
-= Remove Central-attached PV
+= Remove Central-attached PV (version 4.1 and later)
```

Comment:
When I look at the preview of this section <https://65388--ocpdocs-pr.netlify.app/openshift-acs/latest/upgrading/upgrade-operator#remove-central-attached-pv-overview_upgrade-operator>, it troubles me a bit that we don't mention the removal should be done _after_ the operator has upgraded the product. Also, `(version 4.1 and later)` may read ambiguous (whether it is _from_ version 4.1 and newer or _to_ version 4.1 and newer). It could be less ambiguous although lengthier to say `(after upgrading to version 4.1 and later)`.

Reply:
Couple lines down:

"After completing the upgrade to 4.1 you can remove the Central-attached persistent volume claim (PVC) to free up the storage if you do not plan to rollback or restore from earlier RocksDB backups."